libraries/AP_Common/Location.h

```diff
@@ -116,12 +116,12 @@ class Location
 
     bool initialised() const { return (lat !=0 || lng != 0 || alt != 0); }
 
-private:
```

Comment:
You shouldn't need the SCALING_FACTOR's to be public. Use the distance calculation functions instead. Doing the direct manipulation is error prone, and loses out if we ever move to a higher precision one.

Reply:
fixed